src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -512,6 +521,45 @@ The trusted code can store capabilities on the trusted stack when calling out of
 when returning to the same compartment.
 =====
 
+[#ycfg,reftext="ycfg"]
+===== CHERI Capability Encoding (ycfg)
+
+The <<ycfg>> register is used to identify which CHERI capability encoding is used by the platform.
+The capability encoding determines both the in-memory representation of a CHERI capability
+and which CHERI features are available for software's use.
```

Comment:
```suggestion
The capability encoding determines both the in-memory representation of a CHERI capability.
```
I'm not sure we should be promising here that any new "CHERI feature" gets a new value in the ycfg register. For example, adding a future CRelocate extension would be a new CHERI feature but not change the encoding.

Maybe best to be a bit less prescriptive here and just say it determines the in-memory format of the capability?

Reply:
How's this new prose as a compromise?